scripts/seed-probely-db.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+BACKUP=${1:-latest.dump}
```

Comment:
What if we put the probely db dump in `fixtures` with other fake data? (My motivation is simply avoiding clogging up the top level of the repo, which is not a big deal)

Reply:
I can try to get that working! I'll probably make this and the above change in a follow up PR.